### ISSUE TITLE
Missing tags for version 1.0.11 and 1.1.0

### ISSUE THREAD

**Issue by olafmersmann** [tommyod/KDEpy#105]
The public GitHub repository doesn't have tags for the last two versions released on PyPi.

**Comment by tommyod** [comment]
Yes, I changed the deployment so it does not depend on tags. You can download the source from pipy corresponding to the versions.

Are tags more useful? Why do you prefer them?

**Comment by ravigumm** [comment]
@tommyod , Would you please point to the location/URL from where 1.1.0 source can be downloaded from ? Thanks

**Comment by ravigumm** [comment]
How do I get to know this commit level for a particular version 1.1.0 ? Should I go over the git log and understand when the version is modified in source code ? OR Is there a better/easier way from pypi or somewhere else ?

**Comment by tommyod** [comment]
> Should I go over the git log and understand when the version is modified in source code ?

I just looked at the git log. Not optimal, I know - but I don't use tags that actively any more. It requires some digging.

**Comment by ravigumm** [comment]
ok. No problem.
